JenkinsfileTaskTrigger

```diff
@@ -65,44 +65,47 @@ timestamps {
 
                         print "CI_MESSAGE"
                         print CI_MESSAGE
-                        pipelineUtils.flattenJSON('fed', env.CI_MESSAGE)
-                        // Get request vars from scratch messages
-                        env.fed_request_0 = env.fed_request_0 ?: env.fed_info_request_0
-                        env.fed_request_1 = env.fed_request_1 ?: env.fed_info_request_1
-                        // Determine if scratch build
-                        env.isScratch = false
-                        if (env.fed_info_request_0) {
+                        parsedMsg = kojiMessage(message: env.CI_MESSAGE, ignoreErrors: true)
+                        if (parsedMsg.has('info')) {
                             env.isScratch = true
+                            env.request_0 = parsedMsg['info']['request'][0]
+                            env.request_1 = parsedMsg['info']['request'][1]
+                        } else {
+                            env.isScratch = false
+                            env.request_0 = parsedMsg['request'][0]
+                            env.request_1 = parsedMsg['request'][1]
```

Comment:
this code to set env.request and isScratch was used in a couple of files, wouldn't be better to have a function for it?

Reply:
Good suggestion, I can do that